package exceptiondemo2;

public class ExceptionDemo3 {

	public static void main(String[] args) {
		
		
		Vit v=new Vit();
		 v.Add(23, 0);

	}

}

class Vit{
	
	public void Add(int i,int j)
	{
		
		System.out.println(i+j);
	}
	
}